otx/algorithms/classification/tools/classification_sample.py

```diff
@@ -0,0 +1,371 @@
+"""Sample Code of otx training for classification."""
```

Comment:
Is this a sample with both multi-class and multi-label? 
Then I recommend `sample.py` rather than `classification_sample.py`.

Reply:
Yes, it can test all kind of classification. I will revise the file name.

Reply:
I think you forgot this review.

Reply:
Could you look at this again?